tests/mock_tables/config_db.json

```diff
@@ -537,7 +550,7 @@
         "high_mem_alert": "disabled"
     },
     "FEATURE|sflow": {
-        "state": "disabled",
+        "state": "enabled",
```

Comment:
it looks like this change breaks other unit tests

Reply:
i think it is good to revert it back, I will do that.